src/dymaptic.GeoBlazor.Core/Components/LocationService.cs

```diff
@@ -1004,7 +1004,9 @@ public Task<List<AddressCandidate>> AddressToLocations(string url, Address addre
     /// <param name="requestOptions">
     ///     Additional options to be used for the data request 
     /// </param>
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
     public async Task<List<AddressCandidate>> AddressToLocations(string url, Address address,
+#pragma warning restore RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
Suppressing this on purpose. All the possible permutations are there, but the calls between methods are confusing enough for Roslyn to not like what I did here.